src/phantom.js

```diff
@@ -77,7 +77,6 @@ export default class Phantom {
             logger.error(`Could not spawn [${phantomjs.path}] executable. Please make sure phantomjs is installed correctly.`);
             logger.error(error);
             this.kill(`Process got an error: ${error}`);
-            process.exit(1);
```

Comment:
I don't think that the name kill method name should be changed, since it's called as a method on a phantom object.